app/controllers/scholarships_controller.rb

```diff
@@ -42,14 +42,21 @@ def applications
   end
 
   def auto_complete_for_scholarship_title
-		if params[:scholarship][:title].is_integer?
-			@scholarships = [Scholarship.find(params[:scholarship][:title])]
+    term = params[:term] || params[:scholarship][:title]
+		if term.is_integer?
+			@scholarships = [Scholarship.find(term)]
 		else
-	    @scholarships = Scholarship.find(:all, :conditions => ["LOWER(title) LIKE ?", "%#{params[:scholarship][:title].to_s.downcase}%"], :limit => 20)
+	    @scholarships = Scholarship.find(:all, :conditions => ["LOWER(title) LIKE ?", "%#{term.to_s.downcase}%"], :limit => 20)
```

Comment:
Line is too long. [124/80]<br>Use the new Ruby 1.9 hash syntax.<br>Use 2 (not 3) spaces for indentation.<br>Tab detected.